packages/material-ui/src/useAutocomplete/useAutocomplete.js

```diff
@@ -493,6 +493,24 @@ export default function useAutocomplete(props) {
     syncHighlightedIndex();
   });
 
+  if (process.env.NODE_ENV !== 'production') {
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    React.useEffect(() => {
+      if (!inputRef.current || inputRef.current.nodeName !== 'INPUT') {
```

Comment:
Got this warning when using `text area` for autcomplete. Does that mean it's not advisable to use text area when using autocomplete?

Reply:
The Autocomplete is not meant to work with a `<textarea>`. What's your use case?

Reply:
Hi @oliviertassinari, thanks for the reply.
So we actually use this autocomplete for `mention`. It's basically a message with a mention capability, hence we need it as `<textarea>`. Can you share your thoughts about this?

Reply:
@beltjun26 Do you have a codesandbox as an example? How do you position the popup to be right above the mention?